ground/gcs/src/plugins/boards_openpilot/boards_openpilot.pro

```diff
@@ -13,15 +13,12 @@ HEADERS += \
     openpilotplugin.h \
     coptercontrol.h \
     config_cc_hw_widget.h \
-    revolution.h \
-    revomini.h \
```

Comment:
Removing ```revomini.h``` here doesn't fit the pattern.  Too aggressive with the delete or was this file just unused or something?